ui/src/criteria/attribute.tsx

```diff
@@ -18,17 +24,36 @@ interface Config extends CriteriaConfig {
 
 interface Data extends Config {
   selected: Selection[];
+  min: number | undefined;
```

Comment:
These properties will not make sense for every attribute. What's the plan for supporting other attributes? Will this plugin be split out, or is the intent to multiplex different attribute modifiers into this plugin?

If it's the latter, should have a comment here around expectations for these being set, e.g. "min/max may be populated for integer value attributes"

Reply:
Hi CH. This place was modified based on the last review comment from Tim
"I don't think these are working the way you intended. Basic data types like number and string must be defined and the compiler will complain if they're not. That's why isValid is "working" for these variables and not the hints. Instead, you need to define them as number | undefined."

Reply:
done

Reply:
done